lib/gui/components/drive-selector/templates/drive-selector-modal.tpl.html

```diff
@@ -45,7 +50,15 @@ <h4 class="list-group-item-heading">{{ drive.description }} - {{ drive.size | gi
 </div>
 
 <div class="modal-footer">
+  <span ng-show="modal.state.getDrive().system" class="modal-footer-warning">
```

Comment:
I'd rename `modal-footer-warning` to something like `modal-footer-label`, since there is nothing in its definition that is specific to a "warning" element:

```css
.modal-drive-selector-modal .modal-footer-warning {
  padding: 8px;
  display: inline-block;
}
```